spack_repo/local/packages/palace/package.py

```diff
@@ -221,6 +222,7 @@ def cmake_args(self):
             self.define_from_variant("PALACE_WITH_SUNDIALS", "sundials"),
             self.define_from_variant("PALACE_WITH_SUPERLU", "superlu-dist"),
             self.define("PALACE_BUILD_EXTERNAL_DEPS", False),
+            self.define_from_variant("PALACE_MFEM_USE_EXCEPTIONS", "tests"),
```

Comment:
There isn't a palace CMAKE variable that does along with this that enable/disables tests being built? Traditionally if we have the variant there, it would suggest either building or not building tests based on the flag...